src/components/IOList.tsx

```diff
@@ -53,14 +50,15 @@ const IOList = ({
   disableA11y = false
 }: IOListProps) => {
   const { t } = useTranslation()
-  const { getAddress } = useAddressesContext()
   const handleShowAddress = useAddressLinkHandler()
+
   const io = (isOut ? outputs : inputs) as Array<Output | Input> | undefined
 
   if (io && io.length > 0) {
     const isAllCurrentAddress = io.every((o) => o.address === currentAddress)
     const notCurrentAddresses = _(io.filter((o) => o.address !== currentAddress))
       .map((v) => v.address)
+      .filter((v): v is string => v !== undefined)
```

Comment:
some typescript magic to remove `undefined` values so that we don't need to complexify the `map` below.